cmd/podman/networks/rm.go

```diff
@@ -71,6 +71,9 @@ func networkRm(cmd *cobra.Command, args []string) error {
 			fmt.Println(r.Name)
 		} else {
 			setExitCode(r.Err)
+			if networkRmOptions.Force && registry.GetExitCode() == 1 {
```

Comment:
This should only happen if the error is network does not exist, and this is the only error returned.